crates/phactory/api/src/ecall_args.rs

```diff
@@ -18,6 +17,12 @@ pub struct InitArgs {
 
     /// The git commit hash which this binary was built from.
     pub git_revision: String,
+
+    /// Enable geolocation report
```

Comment:
Just a note: when it's stable we are going to enable it by default. It's ok to disable for now.